src/transformers/models/convbert/tokenization_convbert_fast.py

```diff
@@ -46,17 +51,148 @@
 }
 
 
-class ConvBertTokenizerFast(BertTokenizerFast):
+# Copied from transformers.models.bert.tokenization_bert_fast.BertTokenizerFast with bert-base-cased->YituTech/conv-bert-base, Bert->ConvBert all-casing, ConvBertTokenizerFast->BertTokenizerFast
+class ConvBertTokenizerFast(PreTrainedTokenizerFast):
```

Comment:
```suggestion
# Copied from transformers.models.bert.tokenization_bert_fast.BertTokenizerFast with bert-base-cased->YituTech/conv-bert-base, Bert->ConvBert, BERT->ConvBERT
class ConvBertTokenizerFast(PreTrainedTokenizerFast):
```
with all casing the BERT is converted to CONVBERT when its name is ConvBert. Also Bert->ConvBert will do the conversion of the class name :-)